### DISCUSSION TITLE
「セキュリティ設定」-「ページリスト・検索結果」の動作

### DISCUSSION THREAD

**Discussion by funakura** [growilabs/growi#8834] [Help / Questions] [answered]
1. 記事を「自分のみ」で保存する
2.  「セキュリティ設定」-「ページリスト・検索結果」の「自分のみ」のスイッチをON/OFF切替する。
---------------------
「公開」「リンクを知っている人のみ」の説明から、スイッチのON/OFFと非表示 / 表示の関係は以下と理解した。
- スイッチ=ON（表示）
- スイッチ=OFF（非表示）
![image](https://github.com/weseek/growi/assets/43497142/e0307e8a-b07f-41c3-8163-34844407c15b)

以下の場合が、スイッチ=ON（表示）だと思ったが、実際の動作は「非表示」
![image](https://github.com/weseek/growi/assets/43497142/25e39c6c-b47a-4857-836f-cbdda87be3e8)

以下の場合が、スイッチ=OFF（非表示）だと思ったが、実際の動作は「表示」
![image](https://github.com/weseek/growi/assets/43497142/afee19ce-dcd7-42ce-abfa-aa5b8d174874)

要望
-------------------------------------------

スイッチをON/OFFして実際の動作を確認すれば良いだけだが、ON./OFFの設定で毎回悩むためUIに説明を入れるなどしてほしい。

「ON/OFF ＝ 非表示/表示」なのだとは思うが

**Comment by yuki-takei** [comment, marked as answer]
ご要望をいただきありがとうございました。

[v7.2.6](https://github.com/growilabs/growi/releases/tag/v7.2.6) にて、「セキュリティ設定」→「ページリスト・検索結果」の表示設定 UI が改善されました。チェックボックスからドロップダウンメニューへ変更され、各設定値に「常に表示（always displayed）」「常に非表示（always hidden）」というラベルが明示されるようになりました。ON/OFF の意味がひと目でわかる形になっています。
